wordpress-mobile/WordPress-iOS
# Update CocoaPods to 1.6.0

This updates CocoaPods to 1.6.0 which has [many improvements](https://github.com/CocoaPods/CocoaPods/blob/master/CHANGELOG.md#160-2019-02-07) over 1.5.3. Most notably, `pod install` performance is significantly improved.

To test:

- CI checks being green should be enough to test this.
- You can run `rake dependencies` locally if you want to test it out yourself.

Update release notes:
